src/jit/lir.cpp

```diff
@@ -1510,7 +1510,7 @@ LIR::Range LIR::EmptyRange()
 // static
 LIR::Range LIR::SeqTree(Compiler* compiler, GenTree* tree)
 {
-    // TODO(pdg): it would be great to assert that the node has not already been
+    // TODO-LIR: it would be great to assert that the tree has not already been
     // threaded into an order, but I'm not sure that will be practical at this
```

Comment:
I'd like to leave these nice-to-haves maked with `TODO-LIR` s.t. they're easily greppable: these are all things that I intend to follow up soon after the LIR changes are merged. If at that point we end up deciding not to address this in the near term I will remove the `TODO-LIR` marker.